### ISSUE TITLE
Local Graph repository - TyedefNotKnown response formatting is incorrect - missing type name

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#1166]
The error message formating for OMRS-REPOSITORY-0404-004 for the local graph repository is incorrect ie

`
GET http://localhost:8080/servers/Server1/open-metadata/repository-services/users/garygeeke/types/typedef/36f66863-9726-4b41-97ee-714fd0dc6fe4
`
returns
```
{
    "class": "TypeDefResponse",
    "relatedHTTPCode": 404,
    "exceptionClassName": "org.odpi.openmetadata.repositoryservices.ffdc.exception.TypeDefNotKnownException",
    "exceptionErrorMessage": "OMRS-REPOSITORY-404-004 The TypeDef \"getTypeDefByGUID\" (guid = org.odpi.openmetadata.adapters.repositoryservices.graphrepository.repositoryconnector.GraphOMRSMetadataCollection) passed on the Server1 parameter of the {3} operation is not known to the open metadata repository {4}",
    "exceptionSystemAction": "The system is unable to retrieve the properties for the requested TypeDef because the supplied identifier is not recognized.",
    "exceptionUserAction": "The identifier is supplied by the caller.  It may have a logic problem that has corrupted the identifier, or the typedef has been deleted since the identifier was retrieved."
}
```

The actual typedef passed is not added to the log message

**Comment by planetf1** [comment]
cc: @grahamwallis

**Comment by grahamwallis** [comment]
Fixed by PR #2098
